platform/android/MapLibreAndroid/src/main/java/org/maplibre/android/style/types/VariableAnchorOffset.kt

```diff
@@ -0,0 +1,81 @@
+package org.maplibre.android.style.types
+
+import org.json.JSONArray
+import org.json.JSONObject
+
+/**
+ * Represents a collection of anchor offsets.
+ */
+class VariableAnchorOffset(values: Array<AnchorOffset>) {
```

Comment:
Same here - if we decide to use types - let's make it TextVariableAnchorOffsetCollection - same as type in spec and in core code.